PowerEditor/installer/APIs/d.xml

```diff
@@ -0,0 +1,118 @@
+		<KeyWord name="byte" />
+		<KeyWord name="case" />
+		<KeyWord name="cast" />
+		<KeyWord name="catch" />
+		<KeyWord name="cdouble" />
+		<KeyWord name="cent" />
+		<KeyWord name="cfloat" />
+		<KeyWord name="char" />
+		<KeyWord name="class" />
+		<KeyWord name="const" />
+		<KeyWord name="continue" />
+		<KeyWord name="creal" />
+		<KeyWord name="dchar" />
+		<KeyWord name="debug" />
+		<KeyWord name="default" />
+		<KeyWord name="delegate" />
+		<KeyWord name="delete" />
+		<KeyWord name="deprecated" />
+		<KeyWord name="do" />
+		<KeyWord name="double" />
+		<KeyWord name="else" />
+		<KeyWord name="enum" />
+		<KeyWord name="export" />
+		<KeyWord name="extern" />
+		<KeyWord name="false" />
+		<KeyWord name="final" />
+		<KeyWord name="finally" />
+		<KeyWord name="float" />
+		<KeyWord name="for" />
+		<KeyWord name="foreach" />
+		<KeyWord name="foreach_reverse" />
+		<KeyWord name="function" />
+		<KeyWord name="goto" />
+		<KeyWord name="idouble" />
+		<KeyWord name="if" />
+		<KeyWord name="ifloat" />
+		<KeyWord name="immutable" />
+		<KeyWord name="import" />
+		<KeyWord name="in" />
+		<KeyWord name="inout" />
+		<KeyWord name="int" />
+		<KeyWord name="interface" />
+		<KeyWord name="invariant" />
+		<KeyWord name="ireal" />
+		<KeyWord name="is" />
+		<KeyWord name="lazy" />
+		<KeyWord name="long" />
+		<KeyWord name="macro" />
+		<KeyWord name="mixin" />
+		<KeyWord name="module" />
+		<KeyWord name="new" />
+		<KeyWord name="nothrow" />
+		<KeyWord name="null" />
+		<KeyWord name="out" />
+		<KeyWord name="override" />
+		<KeyWord name="package" />
+		<KeyWord name="pragma" />
+		<KeyWord name="private" />
+		<KeyWord name="protected" />
+		<KeyWord name="public" />
+		<KeyWord name="pure" />
+		<KeyWord name="real" />
+		<KeyWord name="ref" />
+		<KeyWord name="return" />
+		<KeyWord name="scope" />
+		<KeyWord name="shared" />
+		<KeyWord name="short" />
+		<KeyWord name="static" />
+		<KeyWord name="struct" />
+		<KeyWord name="super" />
+		<KeyWord name="switch" />
+		<KeyWord name="synchronized" />
+		<KeyWord name="template" />
+		<KeyWord name="this" />
+		<KeyWord name="throw" />
+		<KeyWord name="true" />
+		<KeyWord name="try" />
+		<KeyWord name="typedef" />
+		<KeyWord name="typeid" />
+		<KeyWord name="typeof" />
+		<KeyWord name="ubyte" />
+		<KeyWord name="ucent" />
+		<KeyWord name="uint" />
+		<KeyWord name="ulong" />
+		<KeyWord name="union" />
+		<KeyWord name="unittest" />
+		<KeyWord name="ushort" />
+		<KeyWord name="version" />
+		<KeyWord name="void" />
+		<KeyWord name="volatile" />
+		<KeyWord name="wchar" />
+		<KeyWord name="while" />
+		<KeyWord name="with" />
+		<KeyWord name="__FILE__" />
+		<KeyWord name="__FILE_FULL_PATH__" />
+		<KeyWord name="__MODULE__" />
+		<KeyWord name="__LINE__" />
+		<KeyWord name="__FUNCTION__" />
+		<KeyWord name="__PRETTY_FUNCTION__" />
+		<KeyWord name="__gshared" />
+		<KeyWord name="__traits" />
+		<KeyWord name="__vector" />
+		<KeyWord name="__parameters" />
```

Comment:
the auto-completion file includes the `__` prefixed keywords, which doesn't seem to be in the `langs.model.xml`.  Does the latter need to be updated?